### ISSUE TITLE
Multiple builds failing on Travis

### ISSUE THREAD

**Issue by linduxed** [rmagick/rmagick#229]
Builds for multiple Ruby versions are failing with on the following command:
```
bundle install --jobs=3 --retry=3
```
The command fails when it gets to installing `rmagick`. The error that shows up in the logs is the following:
```
NoMethodError: undefined method `spec' for nil:NilClass
```

**Comment by vassilevsky** [comment]
Do you have any idea why this started? This certainly wasn't an issue before :/

**Comment by linduxed** [comment]
Nope, I've got no idea.

It seems that your `origin/wercker` branch gives [a green build](https://travis-ci.org/rmagick/rmagick/builds/76276783). That branch is basically the tag `RMagick_2-15-4` with almost no changes (37b268c), so that's a good start.

I might check out some commits after that tag and push some branches to make Travis build them. Maybe I'll find the commit that breaks things. Travis-bisect isn't exactly as fast as git-bisect, but I think it should work.

**Comment by linduxed** [comment]
So somewhere between 37b268c and 8614f0c something broke. The diff between those two is as follows:
```patch
diff --git a/README.textile b/README.textile
index f80afba..44bcf9a 100644
--- a/README.textile
+++ b/README.textile
@@ -36,6 +36,8 @@ On Ubuntu, you can run:
 
 bc. sudo apt-get install libmagickwand-dev
 
+*Note to Windows users:* Please make sure that you choose an ImageMagick version that matches the bitness of your Ruby version.
+
 h2(#install). Installing RMagick
 
 h4. Installing via Bundler
```
A documentation change...

Right now, I have no idea what's causing the red builds.

**Comment by mockdeep** [comment]
The error looks familiar to me. I think we may have run into it with a gem upgrade on one of our projects. My guess is one of the dependencies for the project got upgraded and broke our build. Maybe we could try locking them down to an older version to see which one caused it.

**Comment by vassilevsky** [comment]
Fixed with bff778193612edbc9f34dc30fc12d93a2c390532
